static/js/components/CompactPostDisplay.js

```diff
@@ -67,28 +68,47 @@ export class CompactPostDisplay extends React.Component<Props> {
           post.stickied && showPinUI ? "sticky" : ""
         }`}
       >
-        <div className="row title-row">
-          <Link to={postDetailURL(post.channel_name, post.id, post.slug)}>
-            <div className="post-title">
-              <PostTitleAndHostname post={post} />
+        <div className="row post-toprow">
+          <div className="post-toprow-column1">
+            <div className="row title-row">
+              <Link to={postDetailURL(post.channel_name, post.id, post.slug)}>
+                <div className="post-title">
+                  <PostTitleAndHostname post={post} />
+                </div>
+              </Link>
             </div>
-          </Link>
-          {post.url ? (
-            <a href={post.url} target="_blank" rel="noopener noreferrer">
-              <i className="material-icons open_in_new">open_in_new</i>
-            </a>
-          ) : (
-            <i className="material-icons notes">notes</i>
-          )}
-        </div>
-        <div className="row">
-          <div className="authored-by">
-            <div className="author-name">{post.author_name}</div>
-            <div className="date">
-              {formattedDate}
-              {this.showChannelLink()}
+            <div className="row">
+              <div className="authored-by">
+                <div className="author-name">{post.author_name}</div>
+                <div className="date">
+                  {formattedDate}
+                  {this.showChannelLink()}
+                </div>
+              </div>
             </div>
           </div>
+          {post.url ? (
+            <div className="post-toprow-column2">
+              <div className="post-row-top-right">
+                <a href={post.url} target="_blank" rel="noopener noreferrer">
+                  <i className="material-icons open_in_new post-row-top-right overlay-icon">
+                    open_in_new
+                  </i>
+                  {post.thumbnail ? (
+                    <img
+                      ref={`img_${post.id}`}
```

Comment:
what's the ref here for?

Reply:
That's left over from when I was trying to track the card width via state, I'll get rid of it.